ps.go

```diff
@@ -42,12 +42,21 @@ var psCommand = cli.Command{
 			return nil
 		}
 
-		psArgs := context.Args().Get(1)
-		if psArgs == "" {
-			psArgs = "-ef"
+		// [1:] is to remove command name, ex:
+		// context.Args(): [containet_id ps_arg1 ps_arg2 ...]
+		// psArgs:         [ps_arg1 ps_arg2 ...]
+		//
+		psArgs := context.Args()[1:]
+
+		if len(psArgs) > 0 && psArgs[0] == "--" {
```

Comment:
I have colleagues working on refactoring cli with Cobra, we can see how that works out. But for now, I think we can merge this PR.